app/views/users/show.html.haml

```diff
@@ -0,0 +1,6 @@
+%h5 Welcome
+%h6 User Name
+= @user.name
+, #{@user.email}
+
+= link_to 'Edit', edit_user_path(@user)
```

Comment:
This link should not appear unless the current_user == @user.  Need to move this link in to a helper (or view model!)